README.md

```diff
@@ -19,6 +19,36 @@ Runtime for isomorphic javascript libraries (that work in the browser and node.j
 - Set the subscriptionId and token and then run
 - Open index.html file in the browser. It should show the response from GET request on the storage account. From Chrome type Ctrl + Shift + I and you can see the logs in console.
 
+## Architecture Overview
```

Comment:
It would be nice to create a separate folder named documentation or docs and then put the `Archtiecture Overview` over there. README.md should have a link to that document inside the documentation folder.

Reply:
Fixed in https://github.com/Azure/ms-rest-js/pull/154/commits/ff4a56c7502b02ec5cf0089e16bf6345c48c5073.